ucp/__init__.py

```diff
@@ -16,6 +16,18 @@
     logger.debug("Setting env UCX_MEMTYPE_CACHE=n, which is required by UCX")
     os.environ["UCX_MEMTYPE_CACHE"] = "n"
 
+
+# If libucx was installed as a wheel, we must request it to load the library symbols.
+# Otherwise, we assume that the library was installed in a system path that ld can find.
+try:
+    import libucx
+except ImportError:
+    pass
+else:
+    libucx.load_library()
+    del libucx
```

Comment:
Took this directly from here: https://github.com/rapidsai/ucxx/pull/226/files#r1583692142.